portal/models/intervention.py

```diff
@@ -47,14 +51,38 @@ class Intervention(db.Model):
     status_text = db.Column(db.Text)
     public_access = db.Column(db.Boolean, default=True)
     display_rank = db.Column(db.Integer)
+    subscribed_events = db.Column(db.Integer, nullable=False, default=0)
```

Comment:
Is there a better way to make it obvious this a bitfield (eg using binary numbers `0b0`)?